diff --git a/node_modules/@enso-ui/companies/.gitignore b/node_modules/@enso-ui/companies/.gitignore
new file mode 100644
index 0000000..d9d1a55
--- /dev/null
+++ b/node_modules/@enso-ui/companies/.gitignore
@@ -0,0 +1,7 @@
+/node_modules
+/.vscode
+/.idea
+npm-debug.log
+.DS_STORE
+._*
+yarn.lock
\ No newline at end of file
diff --git a/node_modules/@enso-ui/companies/src/bulma/pages/companies/Edit.vue b/node_modules/@enso-ui/companies/src/bulma/pages/companies/Edit.vue
index 0caa203..b096d3b 100644
--- a/node_modules/@enso-ui/companies/src/bulma/pages/companies/Edit.vue
+++ b/node_modules/@enso-ui/companies/src/bulma/pages/companies/Edit.vue
@@ -32,6 +32,28 @@
                             </div>
                         </div>
                     </tab>
+                    <tab keep-alive
+                        id="Comments">
+                        <div class="columns is-centered">
+                            <div class="column is-two-thirds">
+                                <comments :id="companyId"
+                                    type="company"
+                                    @update="count.Comments = $refs.comments.count"
+                                    ref="comments"/>
+                            </div>
+                        </div>
+                    </tab>
+                    <tab keep-alive
+                        id="Documents">
+                        <div class="columns is-centered">
+                            <div class="column is-two-thirds">
+                                <documents :id="companyId"
+                                    type="company"
+                                    @update="count.Documents = $refs.documents.count"
+                                    ref="documents"/>
+                            </div>
+                        </div>
+                    </tab>
                 </template>
             </accessories>
         </div>
@@ -44,6 +66,8 @@ import Accessories from '@enso-ui/accessories/bulma';
 import { Addresses } from '@enso-ui/addresses/bulma';
 import { EnsoForm, FormField } from '@enso-ui/forms/bulma';
 import People from './components/People.vue';
+import { Comments } from '@enso-ui/comments/bulma';
+import { Documents } from '@enso-ui/documents/bulma';
 
 export default {
     name: 'Edit',
@@ -55,6 +79,8 @@ export default {
         Tab,
         Addresses,
         People,
+        Comments,
+        Documents,
     },
 
     computed: {
