diff --git a/node_modules/@enso-ui/discussions/src/bulma/Inputor.vue b/node_modules/@enso-ui/discussions/src/bulma/Inputor.vue
index cdf6c80..f1f3107 100644
--- a/node_modules/@enso-ui/discussions/src/bulma/Inputor.vue
+++ b/node_modules/@enso-ui/discussions/src/bulma/Inputor.vue
@@ -4,18 +4,9 @@
             v-model="message.title"
             :placeholder="i18n('Title...')"
             v-if="title">
-        <quill-editor :options="options"
-            ref="quillEditor"
-            v-model="message.body"/>
-        <form ref="inputForm"
-            @submit.prevent>
-            <input id="file-upload"
-                class="is-invisible"
-                type="file"
-                ref="fileInput"
-                @change="upload($event)"
-                v-if="attachments">
-        </form>
+        <wysiwyg class="has-margin-top-large has-margin-bottom-large"
+             :has-error="false"
+             v-model="message.body"/>
         <div class="has-text-right">
             <a class="button is-small is-rounded"
                 @click="$emit('cancel')">
@@ -42,16 +33,13 @@
 </template>
 
 <script>
-import 'quill/dist/quill.core.css';
-import 'quill/dist/quill.snow.css';
 
-import { quillEditor } from 'vue-quill-editor';
+import Wysiwyg from '@enso-ui/wysiwyg/bulma';
 import { FontAwesomeIcon as Fa } from '@fortawesome/vue-fontawesome';
 import { library } from '@fortawesome/fontawesome-svg-core';
 import { faCheck, faBan } from '@fortawesome/free-solid-svg-icons';
 
 import './mention/mention';
-import './upload/upload';
 import './mention/mention.scss';
 
 library.add(faCheck, faBan);
@@ -59,7 +47,7 @@ library.add(faCheck, faBan);
 export default {
     name: 'Inputor',
 
-    components: { Fa, quillEditor },
+    components: { Fa, Wysiwyg },
 
     inject: ['errorHandler', 'http', 'i18n', 'route'],
 
@@ -68,10 +56,6 @@ export default {
             type: Boolean,
             default: false,
         },
-        attachments: {
-            type: Boolean,
-            default: false,
-        },
         placeholder: {
             type: String,
             required: true,
@@ -95,9 +79,7 @@ export default {
             modules: {
                 toolbar: [
                     ['bold', 'italic', 'strike'],
-                    v.attachments
-                        ? ['blockquote', 'code-block', 'image', 'link']
-                        : ['blockquote', 'code-block', 'link'],
+                    ['blockquote', 'code-block', 'link'],
                     [{ header: 1 }, { header: 2 }],
                     [{ list: 'ordered' }, { list: 'bullet' }],
                     [{ align: [] }],
@@ -106,9 +88,6 @@ export default {
                 mention: {
                     template: item => v.template(item),
                 },
-                upload: {
-                    handler: () => v.openFileBrowser(),
-                },
             },
         },
     }),
@@ -144,23 +123,6 @@ export default {
         taggedUsers() {
             return this.tagged.filter(user => this.message.body.indexOf(this.template(user)) > 0);
         },
-        upload($event) {
-            const Editor = this.$refs.quillEditor.quill;
-            const formData = new FormData();
-
-            formData.append('attachment', $event.target.files[0]);
-
-            this.http.post(this.route('core.discussions.upload'), formData)
-                .then(({ data }) => {
-                    Editor.insertEmbed(
-                        Editor.getSelection().index,
-                        'image',
-                        this.route('core.discussions.showAttachment', data.id),
-                    );
-
-                    this.$refs.inputForm.reset();
-                }).catch(this.errorHandler);
-        },
         avatar(avatarId) {
             return this.route('core.avatars.show', avatarId);
         },
diff --git a/node_modules/@enso-ui/discussions/src/bulma/upload/upload.js b/node_modules/@enso-ui/discussions/src/bulma/upload/upload.js
deleted file mode 100644
index b35e45b..0000000
--- a/node_modules/@enso-ui/discussions/src/bulma/upload/upload.js
+++ /dev/null
@@ -1,11 +0,0 @@
-import Quill from 'quill';
-
-class Upload {
-    constructor(quill, options) {
-        this.quill = quill;
-        this.quill.getModule('toolbar')
-            .addHandler('image', options.handler);
-    }
-}
-
-Quill.register('modules/upload', Upload);
