diff --git a/node_modules/bulma-extensions/bulma-calendar/dist/css/bulma-calendar.sass b/node_modules/bulma-extensions/bulma-calendar/dist/css/bulma-calendar.sass
index cbb2922..f75491f 100644
--- a/node_modules/bulma-extensions/bulma-calendar/dist/css/bulma-calendar.sass
+++ b/node_modules/bulma-extensions/bulma-calendar/dist/css/bulma-calendar.sass
@@ -59,7 +59,7 @@ $calendar-date-padding: .4rem 0 !default
       .calendar-day
         font-size: $size-2
       .calendar-nav-previous-month, .calendar-nav-next-month,
-      .calendar-nav-previous-year, .calendar-nav-next-year,
+      .calendar-nav-previous-year, .calendar-nav-next-year
         flex-basis: auto
         flex-grow: 0
         flex-shrink: 0
